content/en/docs/instrumentation/php/exporters.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Exporters
+weight: 5
```

Comment:
Note to self and @cartermp @svrnm : after this lands we should "space out" the weights so that more pages can fit in between them.

Reply:
ACK

Reply:
Will track via:

- #2431